specification/azuredatatransfer/AzureDataTransfer.Management/Pipeline.tsp

```diff
@@ -106,9 +108,10 @@ interface Pipelines {
   >;
 
   /**
-   * Lists the schemas for the specified connection in a pipeline.
+   * Lists the schemas associated with a specific connection in the Pipeline. This operation is deprecated in version v2025_05_30_preview.
    */
   #suppress "@azure-tools/typespec-azure-core/no-openapi" "Operation Id generated does not match original OpenAPIv2 JSON specification."
+  @removed(Versions.v2025_05_30_preview)
   @action("listSchemas")
   @operationId("ListSchemas_List")
```

Comment:
The operationId comes from the interface name and the operation name.  You can name both to create the operationId,  or you can use `@clientName` to change the operation name in client generation (and there is a similar decorator for changing the operation group (first part of operation id))